src/controllers/update/secure.update.filter.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { NextFunction, Request, Response } from "express";
+
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { IsSecureRegisterKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    const titleExtension: string = "(NOT LIVE)";
+
+    return res.render(config.SECURE_REGISTER_FILTER_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
+      templateName: config.SECURE_REGISTER_FILTER_PAGE,
+      [IsSecureRegisterKey]: appData[IsSecureRegisterKey],
+      titleExtension
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.SECURE_REGISTER_FILTER_PAGE}`);
```

Comment:
I think I left this as `SECURE_REGISTER_FILTER_PAGE` because that was the page it was displaying as it is sharing the same page with the registration journey. Maybe I misunderstood what should go in the log message.

Reply:
Changed this now to UPDATE

Reply:
@stephanie-ch  i have changed this entire line as per your previous suggestion to include `${req.method} ${req.route.path}`